### PR TITLE
remove unuseful yuni string separator

### PR DESCRIPTION
**Pull request by a-zakir**

### CODE REVIEW

File: src/solver/utils/ortools_utils.cpp [hunk 1]
```diff
@@ -214,12 +214,8 @@ static void extract_from_MPSolver(MPSolver* solver,
 
 std::string generateTempPath(const std::string& filename)
 {
-    namespace fs = std::filesystem;
-    std::ostringstream tmpPath;
-    tmpPath << fs::temp_directory_path().string() << Yuni::IO::SeparatorAsString << filename;
-    return tmpPath.str();
+    return (std::filesystem::temp_directory_path() / filename).string().c_str();
```
Comment (a-zakir) [inline src/solver/utils/ortools_utils.cpp]: https://en.cppreference.com/w/cpp/filesystem/path/string

Comment (a-zakir) [reply src/solver/utils/ortools_utils.cpp]: Whenever

**Comment by a-zakir** [comment]
see comments in https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1791

File: src/solver/utils/ortools_utils.cpp [hunk 2]
```diff
@@ -212,14 +215,35 @@ static void extract_from_MPSolver(MPSolver* solver,
...
+        return parentDir;
+    }
+}
 std::string generateTempPath(const std::string& filename)
 {
-    namespace fs = std::filesystem;
-    std::ostringstream tmpPath;
-    tmpPath << fs::temp_directory_path().string() << Yuni::IO::SeparatorAsString << filename;
-    return tmpPath.str();
+    return (CreateRandomSubDir(std::filesystem::temp_directory_path()) / filename).string().c_str();
```
Comment (flomnes) [inline src/solver/utils/ortools_utils.cpp:245]: This changes the behavior. How about keeping the same behavior as previously ?

Comment (a-zakir) [reply src/solver/utils/ortools_utils.cpp:245]: see Pr [reviews](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1791)

Comment (a-zakir) [reply src/solver/utils/ortools_utils.cpp:245]: and we may remove the last 2 commits

**Pull request closed without merging**